languages/php/src/BitwardenClient.php

```diff
@@ -38,23 +67,10 @@ public function __construct(BitwardenSettings $bitwardenSettings)
         $this->secrets = new SecretsClient($this->commandRunner);
     }
 
-    /**
-     * @throws \Exception
-     */
-    public function access_token_login(string $access_token)
+    public function auth(): AuthClient
```

Comment:
Let's make `auth` a property, the same as we have for `projects` and `secrets`.
```
private ?AuthClient $auth;

public function __construct(BitwardenSettings $bitwardenSettings)
{
...
    $this->auth = new AuthClient($this->commandRunner);
}

// Delete auth() method
```

Reply:
I'm not super particular about this kind of thing, but I made it this way because that's how the Rust bindings work: https://github.com/bitwarden/sdk/blob/c04b9a0e950fc79bd14464156030a169f0db14ef/crates/bitwarden-json/src/client.rs#L61

Reply:
Fair. Updated in e20a6b414a60d6bd5cd21f5dcad94ff6bbfe78e6.